src/components/mdx-img/mdx-img.module.css

```diff
@@ -0,0 +1,28 @@
+.root {
+  background-color: var(--token-color-surface-primary);
+  box-shadow: var(--token-surface-base-box-shadow);
+  border-radius: 6px;
+  display: flex;
+  flex-wrap: wrap;
+  overflow: hidden;
+  text-align: center;
+  justify-content: center;
+  margin: 24px auto;
+
+  &.noMargin {
+    margin: 0;
+  }
+
+  &.noBorder {
+    box-shadow: none;
+  }
+}
+
+.img {
+  /* TODO: remove content styles which force !important here */
+  border-radius: 0 !important;
+  background-color: transparent !important;
+  display: block;
+  margin: 0 !important;
+  max-width: 100% !important;
+}
```

Comment:
I think this is an example of why we need to be careful with global element styles. I _feel_ like this responsibility should live with the component.

Reply:
☝️ @ashleemboyer ah yeah excellent clarification - I've been saying `:global` but really intended as "styles leaking in from other components", which don't have to be global. Targeting HTML elements rather than classNames in "wrapper" components has the same effect. And the specificity often ends up being an issue in either case.

Very much agreed on the point of consistency. In the context of this PR, my aim was to add exhaustive (even if duplicative) styles scoped only to Tutorials, based on the design specs I have from Brayden, and to follow-up with a PR that reconciles any differences between docs and learn. Let me know if that makes sense; happy to try to do the `/docs` reconciliation in this PR, but worry that it will balloon the scope quite a bit.

Reply:
Good clarification 👐 I think its okay if we have some style duplicity now given the speed with which we need to move. However we can refactor so that these docs level sweeping styles don't interfere with the Learn components is probably the best path forward? Or we'll just need to be okay with the override debt we're taking on and have a plan to adjust it later.

Reply:
> follow-up with a PR that reconciles any differences between docs and learn

This is totally fine, I just didn't know this was the plan until I asked about it. 😊